server/routes/private.js

```diff
@@ -108,5 +108,26 @@ router.get('/test', async (req, res) => {
     res.send(true);
 });
 
-module.exports = router
+router.post('/api/add-recurring-schedule', async (req, res) => {
+
+
+    // Validate appropriate parameters are passed to view workers at each school 
+    const getWorkersForSchool = Joi.object({
+        schoolId: Joi.number().integer().required(), //The student must specify their school ID in order to view the workers
+    });
+
+    const query = req.query ? req.query : {};
+
+    const schoolId = query.schoolId ? query.schoolId : null;
+
+    const { error } = getWorkersForSchool.validate({schoolId});
+
+    if (!_.isNil(error)) res.send(error);
+
+    // Attempts to insert the available workers at each school 
+    const isSuccessfullyInserted = await schoolHandler.getWorkerIdsForSchool(schoolId);
```

Comment:
Hi Simran,
To avoid confusion, I would rename the variable from `isSuccessfullyInserted` to be `workerIds` as this method returns all the workerIds rather than a boolean value. Additionally, please update the comment on line 117 to say Attempts to get all the workers at the specific school rather than inserting to avoid future confusion.

Reply:
Hi Amy! Thanks for your comments! I've adjusted my code to match your requirements!